nav2_msgs/srv/IsPathValid.srv

```diff
@@ -0,0 +1,5 @@
+#Determine if the current path is still valid
+
+nav_msgs/Path path
+---
+std_msgs/Bool response
```

Comment:
new line at EOF

Also, `response` to something more telling, `is_valid` maybe? 

Also, maybe good to list back the poses in the path that were invalid, might help. I think it would be good to list back the pose index's that failed the check

Reply:
This can also just be bool, no need for std_msgs/Bool. That way its `.response` not `.response.data`

Reply:
Okay, so I would have a new is_valid data type that holds a boolean and the invalid poses.

Reply:
Not a new struct or something, just include multiple fields in the response. `bool` for the fail/success and a `int[]` for the indices

Reply:
okay cool

Reply:
Done